Purchases/Purchasing/IntroEligibilityCalculator.swift

```diff
@@ -94,7 +94,7 @@ private extension IntroEligibilityCalculator {
                     return foundByGroupId
                 }
 
-            let hasIntroductoryPrice = candidate.introductoryPrice != nil
+            let hasIntroductoryPrice = candidate.introductoryDiscount != nil
```

Comment:
> While this is correct, I was wondering whether it'd be a good idea to add a "fake" obsoleted introductoryPrice property to StoreProduct, so that we can also provide an auto-fix for that one.

Ooooh that's actually a great idea! It's not to deal with our own rename, but that would help users greatly transitioning to the new APIs. I'm sure there's other examples of this that we could add.

Reply:
I'll send a PR with that, and maybe we can think of other examples.

Reply:
awesome, thanks Nacho!